MDANSE/Src/MDANSE/Framework/Configurators/TrajectoryFilterConfigurator.py

```diff
@@ -0,0 +1,131 @@
+#    This file is part of MDANSE.
+#
+#    MDANSE is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+import json
+from MDANSE.Framework.Configurators.IConfigurator import IConfigurator
+from MDANSE.Mathematics.Signal import filter_map
+
+
+class TrajectoryFilterConfigurator(IConfigurator):
+    """This configurator allows the application of a filter to the trajectory of atoms in the simulation.
+
+    Attributes
+    ----------
+    _default : str
+        The defaults selection setting.
+    """
+
+    _filter = tuple(filter_map.values())[0]
+
+    @classmethod
+    def filter_default_attributes(cls, filter=_filter):
+        """Get the filter-specific settings dictionary for a filter class.
+
+        Parameters
+        ----------
+        filter :
+            The filter class.
+
+        Returns
+        -------
+
+            The filter settings dictionary
+        """
+        filter.set_defaults()
+        settings_dict = dict()
+        for setting, values in filter.default_settings.items():
+            settings_dict.update({setting: values["value"]})
+        return settings_dict
```

Comment:
```suggestion
        return {setting: values["value"] for setting, values in filter.default_settings.items()}
```
Will do and be faster.